components/salsa-2022/src/runtime/shared_state.rs

```diff
@@ -13,9 +13,6 @@ pub(super) struct SharedState {
     /// Stores the next id to use for a snapshotted runtime (starts at 1).
     pub(super) next_id: AtomicUsize,
 
-    /// Vector we can clone
-    pub(super) empty_dependencies: Arc<[DependencyIndex]>,
-
```

Comment:
Wait, why can't we clone `empty_dependencies` anymore...

Reply:
...this was, admittedly, a pure optimization. But it seemed like it might be a useful one. I guess I don't have any data to back that up. :)

Reply:
It's fine, let's remove it, easy enough to add back later once we have some data.

Reply:
I thought about it for a while and I think I should add it back. I thought it was redundant before, sorry I didn't think carefully here, but now I think it makes sense, there are many queries that have no dependencies, only tracked functions have dependencies? It would be better to have them share an empty dependency. @nikomatsakis